core/pom.xml

```diff
@@ -134,7 +134,7 @@
             <plugin>
                 <groupId>io.swagger.codegen.v3</groupId>
                 <artifactId>swagger-codegen-maven-plugin</artifactId>
-                <version>3.0.23</version>
+                <version>3.0.21</version>
```

Comment:
Could you please add a comment here with a reference to the swagger plugin issue? Just to warn the other dev from upgrading to a more recent version (for whatever reason) until the aforementioned issue is fixed.

Reply:
Sure.